runtime/src/in_mem_accounts_index.rs

```diff
@@ -973,34 +1053,40 @@ impl<T: IndexValue> InMemAccountsIndex<T> {
         startup: bool,
         _flush_guard: &FlushGuard,
     ) -> FlushScanResult<T> {
+        let mut possible_evictions = self.possible_evictions.write().unwrap();
+        if let Some(result) = possible_evictions.get_possible_evictions() {
```

Comment:
Is `flush_scan` going to be called with `current_age` sequentially? Will there be gap between `current_age`?
For example, when processing age 1, assuming bin_mod=1, then it will populate for age 1,2,3,4,5,6. However, the next call skips age 2, and calls with age 3. Then `possible_evictions.get_possible_evictions()` will return the result for age 2. 
Maybe this will never be happen and we always call `flush_scan` with sequential current_age?

Reply:
we always call with sequential age.